frontend/src/app/components/master-page/master-page.component.html

```diff
@@ -1,6 +1,14 @@
+<app-menu *ngIf="servicesEnabled" (loggedOut)="onLoggedOut()"></app-menu>
+
 <ng-container *ngIf="{ val: network$ | async } as network">
-<header *ngIf="headerVisible">
+<header *ngIf="headerVisible" style="position: fixed; width: 100%; z-index: 100;">
```

Comment:
Is the `headerVisible` @Input being used or can it be deleted?

Reply:
It's used in /sponsor and /enterprise